src/main/java/com/startsteps/ecommerceapi/payload/response/ApiResponse.java

```diff
@@ -0,0 +1,12 @@
+package com.startsteps.ecommerceapi.payload.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class ApiResponse {
```

Comment:
Maybe rename this to `MessageResponse` it is more clear what it represents

Reply:
the thing is that I have my message response reserved for the authentication and it has field jwt. would I need to return jwt for the rest or is it better to keep a single responsibility class ApiResponse?

Reply:
it's better to `JwtResponse` and `MessageResponse`. Both are more clear that way

Reply:
I see, thx!